### PR TITLE
Adding LESFMIP and Automating CV generation 

### PR DESCRIPTION
**Pull request by wolfiex**
- Added LESF MIP related items 
- Automated CV creation on push. 
- Reorganisation of the CVs and their structure. 
- Additional metadata added.

### CODE REVIEW

File: .github/libs/__pycache__/checksum_tools.cpython-310.pyc
Comment (durack1) [inline .github/libs/__pycache__/checksum_tools.cpython-310.pyc:1]: You might want to add *.pyc to the .gitignore file

**Review by durack1 — APPROVED** [review]
@wolfiex looks good to me!

File: .github/libs/__pycache__/checksum_tools.cpython-310.pyc
Comment (wolfiex) [reply .github/libs/__pycache__/checksum_tools.cpython-310.pyc:1]: I noted that too

**Review by matthew-mizielinski — APPROVED** [review]
Subject to sorting the conventions string this is good to go

**Comment by matthew-mizielinski** [comment]
Update: some old version headers remain (e.g. CMIP6Plus_experiment_id)

**Comment by wolfiex** [comment]
Versioning implementation functional: awaiting comments or review. 

@durack1 @matthew-mizielinski

**Comment by matthew-mizielinski** [comment]
Nice work @wolfiex. I'm happy for you to tag and release at your leisure.

**Pull request merged**